runtime/caml/fiber.h

```diff
@@ -50,6 +50,9 @@ struct stack_info {
    * Stacks may be unpooled if either the stack size is not 2**N multiple of
    * [caml_fiber_wsz] or the stack is bigger than pooled sizes. */
   struct stack_info** size_bucket;
+#ifdef __OpenBSD__
+  size_t size;
+#endif
```

Comment:
Wouldn't it make sense to store the size in all cases, even on systems that don't currently use it? In particular Stack_ctx_words would not be defined conditionally anymore, making the code/reasoning/debugging simpler.

Reply:
I agree with this. It will be simpler for maintenance. Perhaps we may add a comment saying that this is only used for systems using `MAP_STACK`.